docs/howto/writing-migrations.txt

```diff
@@ -120,7 +120,7 @@ the respective field according to your needs.
         ]
 
 * Edit the first migration file. The generated migration class should look
-  similar to this:
+  similar to this afterwards:
```

Comment:
The current wording is correct. _"The generated migration class should look similar to this"_ so the example shows the unchanged migration, then it describes what needs to be changed.